app/views/admin/users/new/show.html.erb

```diff
@@ -0,0 +1,53 @@
+<style>
+  section {
+    margin-bottom: 1em;
+    box-shadow: none !important;
+  }
+
+  section:first-of-type {
+    margin-top: 1em;
+  }
+</style>
+
+<header>
+  <h2 class="crayons-title">Members</h2>
+  <p><a href="<%= admin_users_path %>">← Go back to All members</a></p>
+</header>
+
+<article
+  class="flex flex-row flex-wrap justify-between content-between"
+  data-controller="user"
+  data-user-tools-component-path-value="<%= admin_user_tools_path(@user.id) %>"
+  data-action="user:tools@document->user#fetchAndOpenTools">
+  <div class="flex flex-column" style="flex-basis: 70%;">
+    <section class="crayons-card" id="profile">
+      <h3 class="crayons-subtitle-2"><%= @user.name %></h3>
```

Comment:
It's more of a design thing, but I think the heading hierarchy is a bit weird here. The biggest heading is "Members" when really the context of the current page is the specific user.

Also the `h1` for the page should really be the one that tells us where we are currently but at the moment it's "Users" up in the header 🙃

Reply:
We could move the info about the current user in the h1, not sure what to keep here though...